crates/ty_python_semantic/resources/mdtest/type_properties/is_subtype_of_given.md

```diff
@@ -0,0 +1,184 @@
+# Constraint implication
+
+```toml
+[environment]
+python-version = "3.12"
+```
+
+This file tests the _constraint implication_ relationship between types, aka `is_subtype_of_given`,
+which tests whether one type is a [subtype][subtyping] of another _assuming that the constraints in
+a particular constraint set hold_.
+
+## Concrete types
+
+For concrete types, constraint implication is exactly the same as subtyping. (A concrete type is any
+fully static type that is not a typevar. It can _contain_ a typevar, though — `list[T]` is
+considered concrete.)
+
+```py
+from ty_extensions import is_subtype_of, is_subtype_of_given, static_assert
+
+def equivalent_to_other_relationships[T]():
+    static_assert(is_subtype_of(bool, int))
+    static_assert(is_subtype_of_given(True, bool, int))
+
+    static_assert(not is_subtype_of(bool, str))
+    static_assert(not is_subtype_of_given(True, bool, str))
```

Comment:
Is `is_subtype_of(T, V)` guaranteed to always be the same as `is_subtype_of(True, T, V)`. If so, maybe that's something you could add to the documentation of `is_subtype_of_given`